public/curriculumAssets/icons/check-circle.svg

```diff
@@ -0,0 +1,4 @@
+<svg class="bi bi-check-circle" width="1em" height="1em" viewBox="0 0 20 20" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```

Comment:
this is a generic icon, let's no maintain our own `svg` images for generic icons.